packages/valory/protocols/tendermint/tests/test_tendermint.py

```diff
@@ -95,7 +95,8 @@ def test_incorrect_error_data_logged(caplog: LogCaptureFixture) -> None:
         error_data=dict(message=1),  # incorrect type
     )
     expected = "Invalid type for dictionary values in content 'error_data'. Expected 'str'. Found '<class 'int'>'."
-    assert expected in caplog.text
+    with caplog.at_level(logging.ERROR, logger=_default_logger.name):
+        assert expected in caplog.text
```

Comment:
hopefully fixes the failure to capture logs seen on ubuntu 3.7 - 3.9 (but not 3.10) and on all macos runs, but none of the windows runs :upside_down_face: